app/assets/javascripts/controllers/network_router/network_router_form_controller.js

```diff
@@ -66,6 +66,10 @@ ManageIQ.angular.app.controller('networkRouterFormController', ['$http', '$scope
     $http.get('/network_router/network_router_networks_by_ems/' + id)
       .then(getNetworkRouterFormByEmsData)
       .catch(miqService.handleFailure);
+
```

Comment:
@himdel I'm a bit confused as to how to refactor this function to use miqService.getProviderTenants - any hints? : )